hack/verify-goimports

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from os import path
+import subprocess
+import sys
+
+from lib.kubernetes import devtools
+
+gopath=os.environ['GOPATH']
```

Comment:
In _theory_ there could be a `$GOPATH` with multiple locations, but not a deal breaker at all. Just a thought.